src/integrationtest/python/JanuaGraphContainer.py

```diff
@@ -0,0 +1,61 @@
+import docker
+from janusgraph_python.driver.ClientBuilder import JanusGraphClient
+from subprocess import Popen, PIPE
+import time
+
+
+class JanusGraphContainer(object):
+    def __init__(self):
+        self.BASE_IMAGE = "florianhockmann/janusgraph-testing:0.3.0"
```

Comment:
We should definitely replace this image with an image from JanusGraph. @pluradj pointed this also out for JanusGraph.Net where I'm also using that image for testing right now.
I think in the beginning we can use the image from janusgraph-docker as soon as we have that ready. Eventually we should probably also make an image available that contains the latest changes from `master` so we can test the drivers always with the current version. Otherwise, we'll only notice problems with the drivers after a release of JanusGraph. But that is really more of a long-term issue.

Reply:
@FlorianHockmann Makes a lot of sense, thanks. In the near-term once janusgraph-docker is ready maybe we could consider updating release docs to indicate that Docker images be published for release candidates so these integration tests can be run as part of the pre-release validation process.